src/routes/taggedNftByOwner.mjs

```diff
@@ -0,0 +1,46 @@
+const MAX_NFTS_PER_PAGE = 100;
+const ATOMIC_ASSET_TAG_VALUE = "atomic-asset";
+
+export const taggedNftByOwner = async (ctx) => {
+  const { page, limit, ownerAddress } = ctx.query;
+  const nodeDb = ctx.nodeDb;
+
+  const parsedPage = page ? parseInt(page) : 1;
+  const parsedLimit = limit
+    ? Math.min(parseInt(limit), MAX_NFTS_PER_PAGE)
+    : MAX_NFTS_PER_PAGE;
+  const offset = parsedPage ? (parsedPage - 1) * parsedLimit : 0;
+
+  try {
+    if (!ownerAddress) {
+      throw Error(`"ownerAddress" param is required`);
+    }
+
+    const result = await nodeDb.raw(
+      `
+        SELECT states.contract_tx_id, states.state
+        FROM states
+        INNER JOIN deployments
+        ON deployments.contract_tx_id = states.contract_tx_id
+        WHERE ? IN (deployments.tag_index_0, deployments.tag_index_1, deployments.tag_index_2, deployments.tag_index_3, deployments.tag_index_4) 
+        AND states.state->>'$.owner' = ?
```

Comment:
I guess this might become slow at some point, so maybe the owner should be moved to a separate column and properly indexed...anyway, let's leave it for now.